### ISSUE TITLE
Ticking Tile Entity after travel through Mystcraft Portal to non-chunk-loaded area

### ISSUE THREAD

**Issue by vizerei** [Railcraft/Railcraft#146]
tl;dr:

Essentially the subject says it all.  The server will crash and have to be restored from backup or the offending Train Detector will have to be deleted via MC Edit to make the world playable again.

Whole story:

Forge: 6.4.20.916
Mystcraft: 0.10.8.0
Railcraft: 8.1.0.0
Other mods: I believe I just use the exact mods that the Forgecraft server uses.  As I only have glimpses at the mod lists from Dire's videos to go by, I could be using different versions and if you need all mods and versions let me know.

The story:

Essentially I had the same setup in three places: the overworld (crash site), and two mystcraft ages for goods trains.  While I was testing the pilot train I would follow it through but every time it would come to the overworld the train would be glitched (all carts existing in the same space) and I wouldn't be able to interact with the minecraft world and would begin falling through the world.  If I exited, I could rejoin and fall back through the world and if I stayed the server crashed throwing out the below error (entire Forge logs available if you need them).  What I eventually found out was that(through MCEdit) the block in question that was the "ticking til entity" was the Train Detect and if I removed that with MCEdit I could load the server back up and join and the train would simply be sitting there waiting for me.  I finally figured out that I hadn't paired the anchor sentinel with the personal anchor after about two days of troubleshooting =.  I haven't tested if this happens without mystcraft portals.

Basic error message:
2013-11-08 17:47:21 [INFO] [STDERR] net.minecraft.util.ReportedException: Ticking tile entity
2013-11-08 17:47:21 [INFO] [STDERR]     at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:674)
2013-11-08 17:47:21 [INFO] [STDERR]     at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:276)
2013-11-08 17:47:21 [INFO] [STDERR]     at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:587)
2013-11-08 17:47:21 [INFO] [STDERR]     at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:484)
2013-11-08 17:47:21 [INFO] [STDERR]     at net.minecraft.server.ThreadMinecraftServer.run(SourceFile:583)
2013-11-08 17:47:21 [INFO] [STDERR] Caused by: java.lang.StackOverflowError
2013-11-08 17:47:21 [INFO] [STDERR]     at com.google.common.collect.MapMakerInternalMap$Segment.get(MapMakerInternalMap.java:2448)
2013-11-08 17:47:21 [INFO] [STDERR]     at com.google.common.collect.MapMakerInternalMap.get(MapMakerInternalMap.java:3434)
2013-11-08 17:47:21 [INFO] [STDERR]     at mods.railcraft.common.carts.LinkageManager.getCartFromLinkageId(LinkageManager.java:99)
2013-11-08 17:47:21 [INFO] [STDERR]     at mods.railcraft.common.carts.LinkageManager.getLinkedCartA(LinkageManager.java:234)
2013-11-08 17:47:21 [INFO] [STDERR]     at mods.railcraft.common.carts.LinkageManager.countLinksRecursive(LinkageManager.java:428)

(the last line will repeat over and over again for at least 1000 lines as my putty session buffer of 2000 lines was used up when cat'ing the entire log)
